app/views/users/edit.html.slim

```diff
@@ -0,0 +1,15 @@
+h1.d-flex.justify-content-center
+  |  Edit Profile
+hr
+.d-flex.justify-content-center
+  = form_with model:@user do |f|
+    .form-group
+      = f.label :email
+      = f.text_field :email , class: "form-control"
+    br
```

Comment:
br izmantošana ir mazliet slikts stils. Bet tas vnk uz nakotni

Reply:
i guess margins are the way to go right? 😄 , es vienkarsi nesmu baigi vel ietraipījies custom css

Reply:
Tas vnk uz nakotni.